iam/snippets/service_accounts_enable.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+// [START iam_enable_service_account]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"golang.org/x/oauth2/google"
+	iam "google.golang.org/api/iam/v1"
+)
+
+// enableServiceAccount enables a service account.
+func enableServiceAccount(w io.Writer, email string) error {
+	client, err := google.DefaultClient(context.Background(), iam.CloudPlatformScope)
```

Comment:
Is there a way to do this with the cloud.google.com/go/iam package? That's generally preferred over the google.golang.org/api/iam packages. I don't see an obvious way to do it at first glance.

Reply:
I don't believe so--it has a completely different set of functions that doesn't include the New(client) function or Enable(ServiceAccount)/Disable(ServiceAccount).